src/components/Card/MucCardContainer.vue

```diff
@@ -0,0 +1,34 @@
+<script setup lang="ts">
+
+</script>
+
+<template>
+  <div class="container card-container">
+    <slot />
+  </div>
+</template>
```

Comment:
```suggestion
<template>
  <div class="container card-container">
    <slot />
  </div>
</template>

<script setup lang="ts">

</script>
```
Consistent order of `template` then `script` then `style`